Fix/documentation installation (#1087)
esm-tools/esm_tools: docs/esm_tools.rst

ESM Tools
=========

`ESM-Tools` includes also a command line tool named ``esm_tools``. It can be used to interact with several parts of our software.

> ``esm_tools`` can be used with two options and two subcommands:

Reviewer: ```suggestion
``esm_tools`` can be used with the following options and subcommands:
```

Reviewer: then we don't need to update this sentence every time we add an option or a subcommand.

+-----------------------------------+-----------------------------------------------------------+
| ``esm_tools`` options             | Description                                               |
+===================================+===========================================================+
| --version                         | Shows the version of the currently used ``esm-tools``.    |
+-----------------------------------+-----------------------------------------------------------+
| --help                            | Prints usage information about ``esm_tools``              |
+-----------------------------------+-----------------------------------------------------------+

Subcommands
^^^^^^^^^^^

+-----------------------------------+--------------------------------------------------+---------------+-------------------------------------------------------+
| ``esm_tools`` subcommand          | Options                                          | Arguments     | Descritpion                                           +
+===================================+==================================================+===============+=======================================================+

> | create-new-config                 | -- help                                          | NAME          | Opens your $EDITOR and creates a new file for NAME    |

Reviewer: ```suggestion
| create-new-config                 | --help                                          | NAME          | Opens your $EDITOR and creates a new file for NAME    |
```
Is the space a typo?

Author: yes

|                                   | -t, --type [component (default)|setup|machine]   |               |                                                       |
+-----------------------------------+--------------------------------------------------+---------------+-------------------------------------------------------+

> | test-state                        | -- help                                          |               | Prints the state of the last tested experiments.      |

Reviewer: ```suggestion
| test-state                        | --help                                          |               | Prints the state of the last tested experiments.      |
```
Is the space a typo?

    Thank you! The new configuration has been saved. Please commit it (and get in touch with the

>     esm-tools team if you need help)!

Reviewer: ```suggestion
    ESM-Tools team if you need help)!
```

Author: I changed it in the documentation. But since this is an example of the output that esm_tools will return, it needs to be changed also in the source code. I'll make a note and change it later.
